src/components/sponsors/style.module.less

```diff
@@ -34,10 +34,46 @@
 
 	a {
 		text-decoration: none;
-		display: block;
+		display: inline-block;
 		color: transparent !important;
 		// Fix IE 11 border
 		background: none !important;
+
+		&:hover img {
+			filter: grayscale(0);
+			opacity: 1;
+		}
+
+		:global(.dark) &:hover img {
+			filter: invert(0) hue-rotate(0deg) grayscale(0)
+				drop-shadow(0 0 0.5px #fff) drop-shadow(0 5px 10px #000)
+				drop-shadow(0 0 20px rgba(255, 255, 255, 0.5));
+		}
+		@media (prefers-color-scheme: dark) {
+			&:hover img {
+				filter: invert(0) hue-rotate(0deg) grayscale(0)
+					drop-shadow(0 0 0.5px #fff) drop-shadow(0 5px 10px #000)
+					drop-shadow(0 0 20px rgba(255, 255, 255, 0.5));
+			}
+		}
+	}
+
+	img {
+		height: 4.5rem;
+		display: block;
+		margin: 0 auto 1rem;
+		opacity: 0.6;
+		transition: all 250ms ease;
+		filter: grayscale(1);
+
+		// Invert based on luminosity
+		@media (prefers-color-scheme: dark) {
+			filter: invert(0.862745) hue-rotate(180deg) grayscale(1);
+		}
+		:global(.dark) & {
+			filter: invert(0.862745) hue-rotate(180deg) grayscale(1);
+			opacity: 1;
+		}
```

Comment:
Before (where's Deno?):
<img width="849" alt="image" src="https://user-images.githubusercontent.com/459878/212196067-32a43012-41bf-4218-a267-74501102a985.png">

After (Ah! There you are!):
<img width="826" alt="image" src="https://user-images.githubusercontent.com/459878/212196189-0ccd809b-528a-4afa-8900-a4b2a4a788a5.png">